addons/arsenal/functions/fnc_buttonLoadoutsShare.sqf

```diff
@@ -0,0 +1,52 @@
+/*
+ * Author: Alganthe
+ * Share selected loadout.
+ *
+ * Arguments:
+ * 0: Arsenal display <DISPLAY>
+ * 1: Button control <CONTROL>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+*/
+#include "script_component.hpp"
+#include "..\defines.hpp"
+
+params ["_display", "_control"];
+
+if !(ctrlEnabled _control) exitWith {};
+
+private _contentPanelCtrl = _display displayCtrl IDC_contentPanel;
+private _contentPanelCursSel = lnbCurSelRow _contentPanelCtrl;
+private _loadoutName = _contentPanelCtrl lnbText [_contentPanelCursSel, 1];
+private _profileName = profileName; // GVAR(center) could be a remote unit
+private _loadoutVar = _profileName + _loadoutName;
+
+private _loadoutIndex = GVAR(sharedLoadoutsVars) find _loadoutVar;
+private _loadoutData = (_contentPanelCtrl getVariable (_loadoutName + str GVAR(currentLoadoutsTab))) select 0;
+
+if (_loadoutIndex > -1) then {
+    GVAR(sharedLoadoutsNamespace) setVariable [_loadoutVar, nil, true];
+    GVAR(sharedLoadoutsVars) = GVAR(sharedLoadoutsVars) - [_loadoutVar];
+
+    _contentPanelCtrl lnbSetPicture [[_contentPanelCursSel, 0], QPATHTOF(data\iconPublicBlank.paa)];
+    _contentPanelCtrl lnbSetValue [[_contentPanelCursSel, 0], 0];
+    [QGVAR(loadoutUnshared), [_contentPanelCtrl, profileName, _loadoutName]] call CBA_fnc_remoteEvent;
+
+} else {
+    GVAR(sharedLoadoutsNamespace) setVariable [_loadoutVar, [_profileName ,_loadoutName , _loadoutData], true];
+    GVAR(sharedLoadoutsVars) pushBackUnique _loadoutVar;
+
+    _contentPanelCtrl lnbSetPicture [[_contentPanelCursSel, 0], QPATHTOF(data\iconPublic.paa)];
+    _contentPanelCtrl lnbSetValue [[_contentPanelCursSel, 0], 1];
+    [QGVAR(loadoutShared), [_contentPanelCtrl, [_profileName ,_loadoutName , _loadoutData]]] call CBA_fnc_remoteEvent;
+};
+
+_control ctrlSetText ( [
+    localize LSTRING(buttonSharePrivateText),
+    localize LSTRING(buttonSharePublicText)
+] select ((_contentPanelCtrl lnbValue [_contentPanelCursSel, 0]) == 1));
+
+publicVariable QGVAR(sharedLoadoutsVars);
```

Comment:
I couldn't find a PVEH for this...
If you don't use it as PVEH you can just put it into sharedLoadoutsNamespace can't you?
You can however use that as a PVEH so you have a eventhandler you can hook when shared loadouts change and know when you need to refresh the list.